gui/wxpython/mapdisp/frame.py

```diff
@@ -549,11 +550,13 @@ def RemoveNviz(self):
 
         # update status bar
         self.statusbarManager.disabledItems = {}
-        self.statusbarManager.SetMode(
+        self.statusbarManager.SetModeWithUpdate(
             UserSettings.Get(group="display", key="statusbarMode", subkey="selection")
         )
+        self.mapWindowProperties.sbItem = UserSettings.Get(
+            group="display", key="statusbarMode", subkey="selection"
+        )
```

Comment:
I didn't test that, but shouldn't setting sbItem trigger SetModeWithUpdate, so the call may not be needed? At least save that in a variable, you are repeating call to UserSettings twice.